controllers/admin/AdminImagesController.php

```diff
@@ -33,17 +35,162 @@ class AdminImagesControllerCore extends AdminController
+                'type' => 'text',
+            ],
+            'PS_IMAGE_GENERATION_METHOD' => [
+                'title' => $this->trans('Generate images based on one side of the source image', [], 'Admin.Design.Feature'),
+                'validation' => 'isUnsignedId',
+                'required' => false,
+                'cast' => 'intval',
+                'type' => 'select',
+                'list' => [
+                    [
+                        'id' => '0',
+                        'name' => $this->trans('Automatic (longest side)', [], 'Admin.Design.Feature'),
+                    ],
+                    [
+                        'id' => '1',
+                        'name' => $this->trans('Width', [], 'Admin.Global'),
+                    ],
+                    [
+                        'id' => '2',
+                        'name' => $this->trans('Height', [], 'Admin.Global'),
+                    ],
+                ],
+                'identifier' => 'id',
+                'visibility' => Shop::CONTEXT_ALL,
+            ],
+            'PS_PRODUCT_PICTURE_MAX_SIZE' => [
+                'title' => $this->trans('Maximum file size of product customization pictures', [], 'Admin.Design.Feature'),
+                'hint' => $this->trans('The maximum file size of pictures that customers can upload to customize a product (in bytes).', [], 'Admin.Design.Help'),
+                'validation' => 'isUnsignedInt',
+                'required' => true,
+                'cast' => 'intval',
+                'type' => 'text',
+                'suffix' => $this->trans('bytes', [], 'Admin.Design.Feature'),
+                'visibility' => Shop::CONTEXT_ALL,
+            ],
+            'PS_PRODUCT_PICTURE_WIDTH' => [
+                'title' => $this->trans('Product picture width', [], 'Admin.Design.Feature'),
+                'hint' => $this->trans('Width of product customization pictures that customers can upload (in pixels).', [], 'Admin.Design.Help'),
+                'validation' => 'isUnsignedInt',
+                'required' => true,
+                'cast' => 'intval',
+                'type' => 'text',
+                'width' => 'px',
+                'suffix' => $this->trans('pixels', [], 'Admin.Design.Feature'),
+                'visibility' => Shop::CONTEXT_ALL,
+            ],
+            'PS_PRODUCT_PICTURE_HEIGHT' => [
+                'title' => $this->trans('Product picture height', [], 'Admin.Design.Feature'),
+                'hint' => $this->trans('Height of product customization pictures that customers can upload (in pixels).', [], 'Admin.Design.Help'),
+                'validation' => 'isUnsignedInt',
+                'required' => true,
+                'cast' => 'intval',
+                'type' => 'text',
+                'height' => 'px',
+                'suffix' => $this->trans('pixels', [], 'Admin.Design.Feature'),
+                'visibility' => Shop::CONTEXT_ALL,
+            ],
+            'PS_HIGHT_DPI' => [
```

Comment:
I guess it's a BC Break to rename this to `PS_HIGH_DPI`? 😅

Reply:
@atomiix It's gonna get removed anyway, I hope. :-)

Reply:
indeed :D